app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,42 @@
+<nav class="navbar navbar-expand-lg bg-light mb-4">
+  <div class="container">
+    <%= link_to "Photogram", root_path, class: "navbar-brand" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
```

Comment:
It's good practice to use Rails path helpers (photos_path, comments_path, etc.) instead of hardcoded paths in your links. This ensures your links remain accurate even if your routes change.